core-client/transports/Cargo.toml

```diff
@@ -44,11 +44,11 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 url = "1.7"
 
-hyper = { version = "0.13", optional = true }
-hyper-tls = { version = "0.4", optional = true }
+hyper = { version = "0.14", features = ["client", "http1", "server"], optional = true }
```

Comment:
the server feature shouldn't be needed here right?

Reply:
You're right, I overlooked that - fixed in https://github.com/paritytech/jsonrpc/pull/628/commits/b05019d5cb88cccdd5a93382aeb8e4f0b1c39f8f